### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by dargmuesli** [prodrigestivill/docker-postgres-backup-local#98] [Q&A] [answered]
Hey there,
is there any changelog to check if upgrading to a new major version introduces breaking changes?
I'd really like to know that to make sure my backups won't break.
If there isn't could there be a changelog added? For example by tagging releases in Git and writing about included changes in GitHub releases.
Thank you, have a great day!

**Comment by prodrigestivill** [comment, marked as answer]
As for now, all git versions are back compatible, it is not considered to release an incompatible update.
Also the version in the docker tags refer to the postgresql version, it is not refered to this application version.
